generate_updateinfo.py

```diff
@@ -241,10 +302,18 @@ def build_updateinfo(src):
                 p_release = "other"
 
             rel_fd[p_release].write('  <update from="%s" status="stable" type="security" version="1.4">\n' % UPDATE_FROM)
```

Comment:
Because this line is left in (and you're adding  `<update>` elements for each severity type) you end up with two opening `<update>` tags for each entry, but only one closing `</update>` tag so in the end the .xml is invalid. It still seems to work somewhat with yum but I get errors. Running the updateinfo.xml through xmllint you'll see though that's it's not valid xml.
for e.g.
````
  <update from="you@your_domain.com" status="stable" type="security" version="1.4">
  <update from="you@your_domain.com" status="stable" type="bugfix" version="1.4">
    <id>CEBA_2015__2396</id>